# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../memtest86/no-scp-install.patch
# Copyright (C) 2019 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- memtest86+-5.01/Makefile.vanilla	2019-01-15 17:56:24.626765041 +0000
+++ memtest86+-5.01/Makefile	2019-01-15 17:56:28.798764994 +0000
@@ -22,7 +22,7 @@
       
 
 all: clean memtest.bin memtest 
-		 scp memtest.bin root@192.168.0.12:/srv/tftp/mt86plus
+		 #scp memtest.bin root@192.168.0.12:/srv/tftp/mt86plus
 
 # Link it statically once so I know I don't have undefined
 # symbols and then link it dynamically so I have full
